lib/vdsm/network/ipwrapper.py

```diff
@@ -302,11 +303,32 @@ def getLinks():
             continue
 
 
+def visible_links():
+    return (_link for _link in getLinks() if not _link.isHidden())
+
+
+def nic_links():
+    return (_link for _link in getLinks() if _link.isNIC())
```

Comment:
Why the leading underscore in `_link`?  To avoid conflict with the importing module?  Then I think `link_` is the conventional way to do it.

Reply:
Yes.